notifier.rst

```diff
@@ -149,6 +149,7 @@ Chatters are configured using the ``chatter_transports`` setting:
 
     # .env
     SLACK_DSN=slack://default/ID
+    SLACK_DSN=slack://default/XXXXXXXXX/XXXXXXXXX/XXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
This might be better than the current situation ... but to me it's the same. How can anybody understand what `XXXX` means? Maybe we need to add a super short comment before each `SLACK_DSN` example explaining what's the `ID` and those `XXXXX`. Thanks!

Reply:
Maybe this comment is helpful (PR updated)

Reply:
I was thinking about something like this:

```rst
# .env
# when using the Slack channel ID
SLACK_DSN=slack://default/ID
# when using Slack webhooks (https://hooks.slack.com/services/T.../B.../X...)
SLACK_DSN=slack://default/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX
```

I don't know if the `ID` is the channel ID; it's only an example. Also, I think we should use the same pattern as the Slack API docs for the different webhook parts. See https://api.slack.com/messaging/webhooks

![image](https://user-images.githubusercontent.com/73419/100064119-64d73500-2e32-11eb-9f49-6487036ae72e.png)

Reply:
I think `slack://default/ID` belongs to a deprecated API that we've removed support for. But Im not sure. 

If Im correct, then `slack://default/ID` cannot be used..

Reply:
It took me hours to get it running using an incoming Webhoook like this:

https://github.com/sonata-project/dev-kit/blob/master/.env.dist

Reply:
The `ID` here refers to the `T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX`  bit, which is the confusing part here.

(and btw, I agree that the Notifier docs are quite a mess. The component evolved a lot after its initial launch, the original document structure no longer fits the broad set of chatters and their different features)